content/stories/fungus-group.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Community Fungus Study
+maybeMetadata:
+  {
+    location: Norfolk,
+    author: Test author,
+    images: [{ src: /images/wildlife-trust-logo.png, alt: "logo" }],
```

Comment:
This should be a markdown list. Something like:

```
images:
  - image:
      - alt: "'My image alt"
      - src: "image/src"
  - image:
      - alt: "another alt"
      - src: "image/source2"
```

Eventually we'll adapt to whatever the CMS encodes as, but I am surprised the parser can handle those special chars.